Author: Adam Di Carlo <aph@debian.org>
Author: Daniel Leidert <daniel.leidert@wgdd.de>
Description: The header in dbpoolx.mod references a wrong Public-ID. This
 patch simply fixes the header comment.
Bug: http://sourceforge.net/tracker/index.php?func=detail&aid=1679712&group_id=21935&atid=384107
Forwarded: yes

--- a/docbook-4.2/dbpoolx.mod
+++ b/docbook-4.2/dbpoolx.mod
@@ -50,7 +50,7 @@
      declaration that uses the public identifier shown below:
 
      <!ENTITY % dbpool PUBLIC
-     "-//OASIS//ELEMENTS DocBook XML Information Pool V4.2//EN"
+     "-//OASIS//ELEMENTS DocBook Information Pool V4.2//EN"
      "dbpoolx.mod">
      %dbpool;
 
--- a/docbook-4.3/dbpoolx.mod
+++ b/docbook-4.3/dbpoolx.mod
@@ -50,7 +50,7 @@
      declaration that uses the public identifier shown below:
 
      <!ENTITY % dbpool PUBLIC
-     "-//OASIS//ELEMENTS DocBook XML Information Pool V4.3//EN"
+     "-//OASIS//ELEMENTS DocBook Information Pool V4.3//EN"
      "dbpoolx.mod">
      %dbpool;
 
--- a/docbook-4.4/dbpoolx.mod
+++ b/docbook-4.4/dbpoolx.mod
@@ -50,7 +50,7 @@
      declaration that uses the public identifier shown below:
 
      <!ENTITY % dbpool PUBLIC
-     "-//OASIS//ELEMENTS DocBook XML Information Pool V4.4//EN"
+     "-//OASIS//ELEMENTS DocBook Information Pool V4.4//EN"
      "dbpoolx.mod">
      %dbpool;
 
--- a/docbook-4.5/dbpoolx.mod
+++ b/docbook-4.5/dbpoolx.mod
@@ -50,7 +50,7 @@
      declaration that uses the public identifier shown below:
 
      <!ENTITY % dbpool PUBLIC
-     "-//OASIS//ELEMENTS DocBook XML Information Pool V4.5//EN"
+     "-//OASIS//ELEMENTS DocBook Information Pool V4.5//EN"
      "dbpoolx.mod">
      %dbpool;
 
